silk/manifest.json

```diff
@@ -0,0 +1,39 @@
+{
+  "display_name": "Silk",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "silk",
+  "metric_prefix": "silk.",
+  "metric_to_check": "silk.system.capacity.free",
+  "creates_events": false,
```

Comment:
I think this should be set to true since the check submits events

Reply:
Good catch, thanks!